### ISSUE TITLE
General architecture

### ISSUE THREAD

**Issue by Glamdring** [18F/analytics-reporter#109]
From the readme I could not fully understand the architecture. Why my guess is:

- each site installs the analytics-reporter
- each site's analytics reporter pushes to the same s3 bucket
- the portal (analytics.usa.gov) reads multiple entries from the s3 bucket

Is that correct? Anyway, could you describe that succinctly in the readme (of one of the two projects)? Or point me to the place where it is described, which I've missed.

**Comment by konklone** [comment]
The best resource for how the project works is [18F's blog post](https://18f.gsa.gov/2015/03/19/how-we-built-analytics-usa-gov/) on how we built it.

The architecture is simpler than your description above. Each participating federal website incorporates a reference to DAP's JavaScript data collection code (which is an [entirely separate project](https://github.com/digital-analytics-program/gov-wide-code)), which collects web traffic into a central Google Analytics Premium account. 

We operate one EC2 server that runs `analytics-reporter` on a cronjob, which connects to that one central account and uploads results to one S3 bucket. The portal, analytics.usa.gov, is getting those entries from the S3 bucket. All of the cross-site aggregation is performed by Google Analytics, before we ever touch it with analytics-reporter.

Does that help? I added a link to our blog post to the analytics.usa.gov repo README in https://github.com/GSA/analytics.usa.gov/commit/fe55eee33668626e7c1faee3c23746f38ae350ba.
